pkg/collector/corechecks/system/disk/disk/disk_nix.go

```diff
@@ -4,149 +4,168 @@
 // Copyright 2016-present Datadog, Inc.
 
 //go:build !windows
+// +build !windows
```

Comment:
Do we still need this build tag syntax?  `// +build ...` 

https://stackoverflow.com/questions/68360688/whats-the-difference-between-gobuild-and-build-directives

Reply:
Not really. I'll remove these build tags